x/staking/types/validator_test.go

```diff
@@ -334,3 +301,22 @@ func TestBondStatus(t *testing.T) {
 	require.Equal(t, BondStatusBonded, Bonded.String())
 	require.Equal(t, BondStatusUnbonding, Unbonding.String())
 }
+
+func mkValidator(tokens int64, shares sdk.Dec) Validator {
+	return Validator{
+		// TODO: use valAddr1
+		OperatorAddress: sdk.ValAddress(pk1.Address().Bytes()).String(),
+		ConsensusPubkey: pk1Any,
+		Status:          Bonded,
+		Tokens:          sdk.NewInt(tokens),
+		DelegatorShares: shares,
+	}
+}
+
+// copy of newValidator. Unfortunately we can't use it because
+// of an import cycle
+func newValidator(t *testing.T, operator sdk.ValAddress, pubKey crypto.PubKey) Validator {
```

Comment:
from where did you copy it, actually?

Reply:
I believe he meant it's a copy of `NewValidator` from `x/staking/teststaking` so maybe better to update the comment to reflect that.

Reply:
Yes. The comment is bad. Thanks for catching. Updated to: 
_Creates a new validators and asserts the error check._